app/models/share.rb

```diff
@@ -10,4 +10,18 @@ class Share < ActiveRecord::Base
   belongs_to :user
 
   validates_presence_of :url, :title, :user
+
+  def color
+    color_category = self.category
+    case color_category
+    when "Article"
+      "success"
```

Comment:
damian had mentioned these options should come in as all caps. for example you can see the filter form and routes for the main calendar page i changed them to all caps. convention purposes i believe.